WordPressAuthenticator/Authenticator/WordPressAuthenticatorDelegateProtocol.swift

```diff
@@ -68,6 +68,14 @@ public protocol WordPressAuthenticatorDelegate: class {
     ///
     func shouldPresentLoginEpilogue(isJetpackLogin: Bool) -> Bool
 
+    /// Indicates the Host app wants to handle and display a given error.
+    ///
+    func shouldDisplayError(_ error: Error) -> Bool
+
+    /// Signals the Host app that there is an error that needs to be handled.
+    ///
+    func handleError(_ error: Error, onCompletion: @escaping (WordPressAuthenticatorResult) -> Void)
```

Comment:
The parameter in the completion block might just be of type UIViewController. After all, host apps would only call this method if they actually want to inject a custom error UI